tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -4274,6 +4274,202 @@ TEST (nnstreamer_capi_singleshot, set_input_info_success_02)
   ml_tensors_info_destroy (out_res);
 }
 
+/**
+ * @brief Test NNStreamer single shot (tflite)
+ * @detail run the `ml_single_invoke_dynamic` api works properly.
+ */
+TEST (nnstreamer_capi_singleshot, invoke_dynamic_success)
+{
+  ml_single_h single;
+  int status;
+  ml_tensors_info_h in_info, out_info;
+  ml_tensors_data_h input, output;
+
+  unsigned int tmp_count;
+  ml_tensor_type_e tmp_type = ML_TENSOR_TYPE_UNKNOWN;
+  ml_tensor_dimension tmp_dim;
+
+  const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
+  gchar *test_model;
+
+  /* supposed to run test in build directory */
+  if (root_path == NULL)
+    root_path = "..";
+
+  /* dynamic dimension supported */
+  test_model = g_build_filename (root_path, "tests", "test_models", "models",
```

Comment:
There is no memory leak for 'test_model' without freeing it (at 4302). However, the static analysis tools we are using would complain about it.

Reply:
I'll apply for all the same cases at another pr.